chart/values.yaml

```diff
@@ -5,6 +5,9 @@ image:
 service:
   enabled: true                              # Creates a Kubernetes Service for the unleash-service
 
+subscription:
+  pubsubTopic: "sh.keptn.event.action.triggered" # Sets the events the service subscribes to
```

Comment:
Is this parameter documented in the Helm chart readme?

Reply:
No just added it